config/default/manager_image_patch.yaml

```
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: ghcr.io/tinkerbell/cluster-api-provider-tinkerbell:dev
+      - image: tinkerbell/cluster-api-provider-tinkerbell-amd64:dev
```

We'll build a multi-arch image, so this can probably just be
```suggestion
      - image: tinkerbell/cluster-api-provider-tinkerbell:dev
```